src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -169,21 +171,45 @@ static void resultFooter(RoutingContext ctx, Integer count, String diagnostic) {
 
   Future<Void> getReportTitles(Vertx vertx, RoutingContext ctx) {
     try {
+      PgCqlQuery pgCqlQuery = PgCqlQuery.query();
```

Comment:
It could be moved out. It could be a little more difficult to read and the performance penalty is minimal. Since the query handling - eventually could be tenant specific in the future,it might not work anyway.

Reply:
OK, I am convinced. Leave it.